packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml

```
@@ -2290,7 +2295,7 @@ processors:
         "257": CAA
         "32768": TA
         "32769": DLV
-      if: ctx.event?.action == "DnsRequest" && ctx.crowdstrike?.RequestType != null && !ctx.crowdstrike.RequestType.isEmpty()
+      if: ctx.event?.action != null && ctx.event?.action.contains("DnsRequest") && ctx.crowdstrike?.RequestType != null && !ctx.crowdstrike.RequestType.isEmpty()
```

```suggestion
      if: ctx.event?.action != null && ctx.event.action.contains("DnsRequest") && ctx.crowdstrike?.RequestType != null && !ctx.crowdstrike.RequestType.isEmpty()
```
`ctx.dns?.type == "query" && ctx.crowdstrike?.RequestType != null && !ctx.crowdstrike.RequestType.isEmpty()`?